wasmtime-wasi/sandboxed-system-primitives/src/config.h

```diff
@@ -20,15 +20,15 @@
 #define CONFIG_HAS_ARC4RANDOM_BUF 0
 #endif
 
-// On Linux, prefer to use getentropy, though it isn't available in
+// On Linux, prefer to use getrandom, though it isn't available in
 // GLIBC before 2.25.
```

Comment:
Is this still the correct version?

Reply:
[yes](https://sourceware.org/git/?p=glibc.git;a=blob;f=NEWS;h=0e4c57f27335d4d62580765aec74fce538459821;hb=HEAD#l1870)